[DataGrid] @material-ui/styles not found when opening codesandbox demo

I try to use DataGrip with version 5, but it shows an error:
```
./node_modules/@material-ui/data-grid/dist/index-esm.js
Module not found: Can't resolve '@material-ui/styles' in '/.../node_modules/@material-ui/data-grid/dist'
```
## Current Behavior 😯

[Follow the example on next.material-ui](https://codesandbox.io/s/94ivi?file=/demo.js)

## Your Environment 🌎

```
  "dependencies": {
    "@emotion/react": "^11.4.0",
    "@emotion/styled": "^11.3.0",
    "@material-ui/core": "^5.0.0-beta.2",
    "@material-ui/data-grid": "^4.0.0-alpha.34",
    "@material-ui/icons": "^5.0.0-beta.1",
    "@material-ui/lab": "^5.0.0-alpha.41",
    "lodash": "^4.17.21",
    "material-ui-popup-state": "^1.9.0",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-router-dom": "^5.2.0",
    "react-scripts": "^1.1.5"
  }
```

--- a/docs/src/pages/components/data-grid/getting-started/getting-started.md
+++ b/docs/src/pages/components/data-grid/getting-started/getting-started.md
@@ -18,14 +18,15 @@ npm install @material-ui/data-grid
 yarn add @material-ui/data-grid
 ```
 
-The grid has a peer dependency on Material-UI core components. If you are not already using Material-UI in your project, you can install it with:
+The grid has two peer dependencies on Material-UI components.
+If you are not already using Material-UI in your project, you can install it with:
 
 ```sh
 // with npm
-npm install @material-ui/core
+npm install @material-ui/core @material-ui/styles
 
 // with yarn
-yarn add @material-ui/core
+yarn add @material-ui/core @material-ui/styles
 ```
 
 ## Quick start

--- a/packages/grid/data-grid/package.json
+++ b/packages/grid/data-grid/package.json
@@ -46,6 +46,7 @@
   },
   "peerDependencies": {
     "@material-ui/core": "^4.9.12 || ^5.0.0-beta.0",
+    "@material-ui/styles": "^4.9.10 || ^5.0.0-beta.0",
     "react": "^17.0.0"
   },
   "setupFiles": [

--- a/packages/grid/x-grid/package.json
+++ b/packages/grid/x-grid/package.json
@@ -47,6 +47,7 @@
   },
   "peerDependencies": {
     "@material-ui/core": "^4.9.12 || ^5.0.0-beta.0",
+    "@material-ui/styles": "^4.9.10 || ^5.0.0-beta.0",
     "react": "^17.0.0"
   },
   "setupFiles": [

